crates/compiler/builtins/bitcode/src/main.zig

```diff
@@ -253,7 +253,9 @@ test "" {
 
 // Export it as weak incase it is already linked in by something else.
 comptime {
-    @export(__muloti4, .{ .name = "__muloti4", .linkage = .Weak });
+    if (builtin.target.os.tag != .windows) {
```

Comment:
Can we remove this `__muloti4`? We initially added it as a workaround. It theoretically shouldn't be needed, but is always linked. Not sure if updates to the build process have changed that. I know it has occasionally caused minor linking issues for me.